src/Components/Filter/Filter.css

```diff
@@ -0,0 +1,57 @@
+ /* FILTERS BAR */
+  
+ .filters {
+    padding: 1rem;
+  }
+  
+  .filter-heading {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    border-bottom: 2px solid var(--primary);
+  }
+  
+  .filters input{
+    margin-right: 0.2rem;
+  }
+
+  .clear {
+    color: var(--buttons);
+    font-weight: bolder;
+    font-size: larger;
+    text-shadow: 1px 1px 2px  var(--font);
+    cursor: pointer;
+  }
+  
+  .filter-head {
+    margin: 1rem 0rem;
+  }
+  
+  .filter-category {
+    text-align: left;
+  }
+  
+  .filter-category:hover{
```

Comment:
why giving cursor pointer on hover ?
you can just give the cursor pointer in `.filter-category` it will work same.

Reply:
Got it, will fix it.